src/main/java/nl/hsac/fitnesse/junit/allure/JUnitAllureFrameworkListener.java

```diff
@@ -188,25 +188,26 @@ private void recordTestResult(Description description) {
     }
 
     private void processAttachments(Throwable ex, List<Pattern> patterns) {
-
         for (Pattern pattern : patterns) {
-            Matcher patternMatcher = pattern.matcher(ex.getMessage());
-            if (patternMatcher.find()) {
-                String filePath = HsacFitNesseRunner.FITNESSE_RESULTS_PATH + "/" + patternMatcher.group(1);
-                String attName;
-                String type;
-                String ext = FilenameUtils.getExtension(Paths.get(filePath).toString());
-                if (ext.equalsIgnoreCase(SCREENSHOT_EXT)) {
-                    attName = "Page Screenshot";
-                    type = "image/png";
-                } else if (ext.equalsIgnoreCase(PAGESOURCE_EXT)) {
-                    attName = "Page Source";
-                    type = "text/html";
-                } else {
-                    attName = "Attachment";
-                    type = "text/html";
+            if (null != ex.getMessage()) {
```

Comment:
It seems this check can be places outside the `for`, the pattern does not change whether a message is available